djangocms_text_ckeditor/fields.py

```diff
@@ -12,28 +15,69 @@
     pass
 
 
+class HTMLFormField(CharField):
+
+    widget = TextEditorWidget
+
+    def __init__(self, *args, **kwargs):
+        conf = kwargs.pop('configuration', None)
+
+        if conf:
+            widget = TextEditorWidget(configuration=conf)
+        else:
+            widget = None
+        kwargs.setdefault('widget', widget)
+        super(HTMLFormField, self).__init__(*args, **kwargs)
+
+    def clean(self, value):
+        value = super(HTMLFormField, self).clean(value)
+        return clean_html(value, full=False)
+
+
 class HTMLField(models.TextField):
     configuration = None
 
+    if LTE_DJANGO_1_7:
+        __metaclass__ = models.SubfieldBase
```

Comment:
@cyberstar It's not 3.3 and 3.4 compatible.. and both were removed from test matrix.
I'll add it like f1676b8532ccc8e200be0b64eb791b20b1793ae1 just in case